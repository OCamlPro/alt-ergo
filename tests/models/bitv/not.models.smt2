(set-logic BV)
(set-option :produce-models true)

(declare-const x (_ BitVec 2))
(assert (= (bvnot x) #b00))
(check-sat)
(get-model)
