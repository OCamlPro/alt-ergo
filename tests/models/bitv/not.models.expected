
unknown
(
  (define-fun x () (_ BitVec 2) #b11)
)
